arch/arm64/boot/dts/rockchip/rk3328.dtsi

```diff
@@ -461,12 +461,12 @@
 		#size-cells = <0>;
 		clocks = <&cru SCLK_SPI>, <&cru PCLK_SPI>;
 		clock-names = "spiclk", "apb_pclk";
-		dmas = <&dmac 8>, <&dmac 9>;
-		#dma-cells = <2>;
-		dma-names = "tx", "rx";
+		/* dmas = <&dmac 8>, <&dmac 9>; */
```

Comment:
I believe that you should be able to remove these keys too from rock64.dts. Or maybe we should just "zero" them?